prime-angular-frontend/src/app/shared/components/forms/college-certification-form/college-certification-form.component.html

```diff
@@ -35,7 +35,23 @@
                       [ngTemplateOutlet]="collegeLicences"></ng-container>
       </div>
 
-      <ng-container *ngIf="collegeCode.value">
+      <div class="col-sm-12">
+        <ng-container *ngIf="!condensed && collegeCode.value && collegeCode.value === CollegeLicenceClassEnum.BCCNM">
```

Comment:
Looks like you can drop one of the `college.value` and instead have `collegeCode?.value === CollegeLicenceClassEnum.BCCNM`

Reply:
very true, probably had my brain in a  knot from all the conditionals i was doing.